src/components/SliderInput.js

```diff
@@ -1,34 +1,30 @@
 import React from 'react';
-import Slider from 'rc-slider';
 
-function RangeInput(props) {
+function SliderInput(props) {
```

Comment:
Just FYI I renamed this to `SliderInput` in `master` mainly because the user is not selecting a range, they're choosing a single value.